WcaOnRails/config/locales/ru.yml

```diff
@@ -1402,11 +1403,11 @@ ru:
     nearby_competitions:
       #original_hash: 241801a
       competitions:
-        other: '%{count} WTF'
-        many: '%{count} соревнований'
-        few: '%{count} соревнования'
-        one: '%{count} соревнование'
         zero: Нет соревнований
+        one: '%{count} соревнование'
+        few: '%{count} соревнования'
+        many: '%{count} соревнований'
+        other: '%{count} WTF'
```

Comment:
None of it, probably a bug in rails-i18n, or some requirement to have "other" that we don't know of.
`i18n-spec` (that checks for missing pluralization keys), uses [this](https://github.com/tigrish/iso/blob/29a27fff5202b5db3ac97a42867b8a2f508b3687/data/iso-639-1.yml#L215) set of keys for Russian too.